packages/react/src/patterns/blocks/LeadSpaceBlock/LeadSpaceBlock.js

```diff
@@ -0,0 +1,69 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings } from 'carbon-components';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import {VideoPlayer} from "../../../components/VideoPlayer";
+import { LinkList } from '../../sub-patterns/LinkList';
+import { Button} from 'carbon-components-react';
+import { ContentBlock } from '../../sub-patterns/ContentBlock';
+import {ImageWithCaption} from "../../../components/ImageWithCaption";
+import Layout from "../../sub-patterns/Layout/Layout";
+
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+const _renderMedia = (type, data) => {
```

Comment:
Weighing in, just wondered if we define this as an inner (private) component. So we can do: `<Media type={mediaType} data={mediaData} />`

Reply:
yeah we had talks of creating a media component that would handle the switching between image and video...what happened to that @jeffchew

Reply:
There's some functional specs around this but we haven't done this yet, we wanted to get this media switching out quickly and waiting for visual/functional specs would have taken too long. It's called the media manager, and is an iceboxed item (you can find it in the icebox pipeline)